### PR TITLE
Fix autocomplete clear button's position when floating label is used

### LINKED ISSUE #539: paper-autocomplete + floating label + allowClear=true
Running against alpha6.

If I use paper-autocomplete with the floating label option and allowClear=true the reset button appears beneath the autocomplete input rather than inline with it as it seems like it should.

screen shot 2016-10-31 at 3 51 02 pm

The template code used for this is:
```
{{#paper-autocomplete
        label="Business Name"
        selected=selectedCluster
        search=(perform searchClusters)
        labelPath="name"
        searchMessage=false
        allowClear=true
        onSelectionChange=(action 'selectionChanged') as |cluster autocomplete|}}
    <span class="item-title">
      <span><strong>{{paper-autocomplete-highlight searchText=autocomplete.searchText label=cluster.name}}</strong> &mdash; {{cluster.address}} ({{cluster.cluster_id}})</span>
    </span>
{{/paper-autocomplete}}
```

### PR DESCRIPTION
**Pull request by xomaczar**
This closes #539

### CODE REVIEW

**Comment by miguelcobain** [comment]
@xomaczar thanks for the PR.
Can you summarize this change?
What exactly happened to the backport? Is it just a copy-pasted version of latest AM styles for autocomplete?

**Comment by xomaczar** [comment]
@miguelcobain Sure, no problem - tried to comment as much as I could.
1. The backported autocomplete style is simply a copy-pasted version of the latest AM styles @1.1.1
2. In the new AM autocomplete style the `md-autocomplete-suggestions-containers` height/max-height properties were removed, so to make it compatible I added those back. (Later on we can make the necessary autocomplete changes in `paper-autocomplete/component.js` and remove height/max-height from css - match AM style)

Hope this helps.

**Comment by xomaczar** [comment]
@miguelcobain does that explain my changes?

**Comment by xomaczar** [comment]
can you merge this PR?

**Comment by xomaczar** [comment]
rebased
